spec/features/jobs/user_sees_tags_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
```

Comment:
Really like how you wrote all tests at once. Would liked to have seen some sad path testing here.